view-creator-framework/src/test/java/org/hypertrace/core/viewcreator/pinot/PinotUtilsTest.java

```diff
@@ -101,6 +101,7 @@ public void testCreatePinotSchema() {
         new DateTimeFormatSpec(dateTimeFieldSpec.getFormat()).getColumnUnit());
     assertEquals(DataType.LONG, dateTimeFieldSpec.getDataType());
     assertEquals(0L, dateTimeFieldSpec.getDefaultNullValue());
+    assertEquals(DataType.BOOLEAN, pinotSchema.getDimensionSpec("is_external").getDataType());
```

Comment:
So Pinot type for `is_external` avro field comes out to BOOLEAN now because of the new version of Pinot lib, and with older version it would have been converted to String type?